tools/go-cli/go-whisk-cli/commands/package.go

```diff
@@ -76,11 +40,19 @@ var packageBindCmd = &cobra.Command{
     PreRunE: setupClientConfig,
     RunE: func(cmd *cobra.Command, args []string) error {
         var err error
-        if len(args) != 2 {
-            whisk.Debug(whisk.DbgError, "Invalid number of arguments %d; args: %#v\n", len(args), args)
-            errStr := fmt.Sprintf("Invalid number of arguments (%d) provided; either the package name or the binding name is missing", len(args))
-            werr := whisk.MakeWskError(errors.New(errStr), whisk.EXITCODE_ERR_GENERAL, whisk.DISPLAY_MSG, whisk.DISPLAY_USAGE)
-            return werr
+
+        if len(args) < 2 {
+            whisk.Debug(whisk.DbgError, "Package bind command must have at least one argument\n")
+            errMsg := fmt.Sprintf("Invalid argument(s). A package name, and binding name are required.")
+            whiskErr := whisk.MakeWskError(errors.New(errMsg), whisk.EXITCODE_ERR_GENERAL,
+                whisk.DISPLAY_MSG, whisk.DISPLAY_USAGE)
+            return whiskErr
+        } else if len(args) > 2 {
+            whisk.Debug(whisk.DbgError, "Package bind command must not have more than two arguments\n")
+            errMsg := fmt.Sprintf("Invalid argument(s): %s.", strings.Join(args[2:], ", "))
```

Comment:
How about including the reason why the argument are invalid (i.e. too many arguments)? Otherwise, one might think the arguments might just be misspelled or something like that.